workflow/log-monitoring/transactions.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Transforming Logs into Transactions
+sidebarTitle: Transactions
+---
+
+Transactions traverse across your system and aggregate your log data with other data that exists outside of the log you are already filtering. This feature is useful when you need to identify broader patterns within your infrastructure that are related to the log you are actively filtering. 
```

Comment:
This section is off. All of this data comes from the logs actually. Each attribute displayed here are attributes in the log json. 

What transactions does is provide the user the ability to run aggregations on top of their logs across all attributes collected. The benefit is being able to build a transactional view of the logs, chosing which attributes to group by and which to run aggregations on top of.